forui/lib/src/widgets/button/button.dart

```diff
@@ -99,6 +99,18 @@ class FButton extends StatelessWidget {
           label: label,
         );
 
+  /// Creates a [FButton] that contains only an [icon]
+  FButton.icon({
+    required this.onPress,
+    required Widget icon,
```

Comment:
Should we rename this to `child` @kawaijoe ? The [usage](https://github.com/forus-labs/forui/pull/138/files#diff-36884b437c4b1ac665bf24853e5fffe851ad98cd8753c981f6690c7fb241e798R146-R152) seems a little weird.

Reply:
I think `child` makes sense here